tasks/system_probe.py

```diff
@@ -173,22 +173,6 @@ def build_dev_docker_image(ctx, image_name, push=False):
             ctx.run(push_cmd.format(image_name=image_name))
 
 
-@task
-def codegen(ctx):
-    """codegen handles retrieving the easyjson dependency and rebuilding
-    the easyjson files
-    """
-
-    ctx.run("go get -u github.com/mailru/easyjson/...")
```

Comment:
should we replace that by a command to handle the protobuf generation ?

Reply:
humm, good point, but since we're using the `process-agent` messages shouldn't this be left there?

Reply:
👍